kubeflow/trainer/api/trainer_client.py

```
@@ -12,576 +12,208 @@
+
+        Raises:
+            TimeoutError: Timeout to get a TrainJob.
+            RuntimeError: Failed to get a TrainJob.
+        """
+        return self.backend.get_job_logs(name=name, follow=follow, step=step)
 
     def wait_for_job_status(
         self,
         name: str,
-        status: Set[str] = {constants.TRAINJOB_COMPLETE},
+        status: set[str] = {constants.TRAINJOB_COMPLETE},
```

Ruff (0.13.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)